app/subscriber/src/features/hooks/useSubMediaGroups.ts

```diff
@@ -118,14 +119,7 @@ export const useSubMediaGroups = (
       newSubGroups = newSubGroups.filter((sg) => !userInfo?.mediaTypes.includes(sg.key));
     }
     setSubMediaGroups(newSubGroups);
-  }, [
-    mediaTypeSeriesLookup,
-    mediaTypeSourceLookup,
-    mediaTypes,
-    series,
-    sources,
-    userInfo?.mediaTypes,
-  ]);
+  }, [mediaTypeSeriesLookup, mediaTypeSourceLookup, mediaTypes, series, sources, userInfo]);
```

Comment:
This looks more dramatic than it is, just `userInfo?.mediaTypes` -> `userInfo` for deps array